### PR TITLE
wex balancing

### PR DESCRIPTION
**Pull request by end-me-please**
its too op

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [X] I have not read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [X] I have ensured that my code compiles, if applicable.
- [X] I have ensured that any new features in this PR dysfunction correctly in-game, if applicable.

### CODE REVIEW

**Comment by end-me-please** [comment]
to keep the  designers on their toes

**Comment by Breadyboi69** [comment]
seems legit. this drastic change should be included in v8 at once.

File: core/src/mindustry/content/Blocks.java [hunk 1]
```diff
@@ -2713,7 +2713,7 @@ public static void load(){
         waterExtractor = new SolidPump("water-extractor"){{
             requirements(Category.production, with(Items.metaglass, 30, Items.graphite, 30, Items.lead, 30, Items.copper, 30));
             result = Liquids.water;
-            pumpAmount = 0.11f;
+            pumpAmount = 0.105f;
```
Comment (Anuken) [inline core/src/mindustry/content/Blocks.java:2716]:
```suggestion
            pumpAmount = -0.105f;
```

**Comment by Def-Try** [comment]
vater intractor real

File: core/src/mindustry/content/Blocks.java [hunk 2]
```diff
@@ -2713,7 +2713,7 @@ public static void load(){
         waterExtractor = new SolidPump("water-extractor"){{
             requirements(Category.production, with(Items.metaglass, 30, Items.graphite, 30, Items.lead, 30, Items.copper, 30));
             result = Liquids.water;
```
Comment (LixieWulf) [inline core/src/mindustry/content/Blocks.java:2715]:
```suggestion
            result = Liquids.slag;
```

**Comment by Breadyboi69** [comment]
this is beautiful

File: core/src/mindustry/content/Blocks.java (hunk 2 above)
Comment (MEEPofFaith) [inline core/src/mindustry/content/Blocks.java:2715]:
```suggestion
            result = Liquids.neoplasm;
```

**Comment by Def-Try** [comment]
incredible

File: core/src/mindustry/content/Blocks.java [hunk 3]
```diff
@@ -2713,7 +2713,7 @@ public static void load(){
         waterExtractor = new SolidPump("water-extractor"){{
             requirements(Category.production, with(Items.metaglass, 30, Items.graphite, 30, Items.lead, 30, Items.copper, 30));
```
Comment (Leo-MathGuy) [inline core/src/mindustry/content/Blocks.java:2714]:
```suggestion
            requirements(Category.production, with(Items.dormantCyst, 30, Items.graphite, 30, Items.lead, 30, Items.copper, 30));
```

**Comment by Anuken** [comment]
too balanced

**Comment by MEEPofFaith** [comment]
real

**Pull request closed without merging**